pkg/apis/policy/v1alpha1/propagation_types.go

```diff
@@ -443,6 +451,26 @@ const (
 	DynamicWeightByAvailableReplicas DynamicWeightFactor = "AvailableReplicas"
 )
 
+// PreemptionBehavior describes whether and how to preempt resources that are
+// claimed by lower-priority PropagationPolicy(ClusterPropagationPolicy).
+// +enum
+type PreemptionBehavior string
+
+const (
+	// PreemptLowerPriority means that preemption is allowed as per Priority.
+	// If it is applied to a PropagationPolicy, it can preempt any resource as
+	// per Priority, regardless of whether it has been claimed by a PropagationPolicy
+	// or a ClusterPropagationPolicy, as long as it can match the rules defined
+	// in ResourceSelector.
+	// If it is applied to a ClusterPropagationPolicy, it can only preempt from
+	// ClusterPropagationPolicy, and from PropagationPolicy is not allowed.
```

Comment:
I think it is necessary to add a situation where low-priority pp can preempt high-priority cpp.

Reply:
PreemptLowerPriority -> PreemptLowerGrade
Grade：high-priority pp > low-priority pp > high-priority cpp > low-priority cpp

Reply:
> I think it is necessary to add a situation where low-priority pp can preempt high-priority cpp.

Added in comments:
```
In addition, if a resource has already been claimed by a ClusterPropagationPolicy, the PropagationPolicy can still preempt it without considering Priority.
```